testing/tests/DevExpress.viz.sparklines/bulletTooltip.tests.js

```diff
@@ -47,77 +80,91 @@ const environment = {
         });
         this.renderer = new vizMocks.Renderer();
         this.$container = $(createTestContainer('#container'));
-        this.createBullet = function(options) {
-            return this.$container.dxBullet($.extend(true, {
-                tooltip: {
-                    enabled: true
-                }
-            }, options)).dxBullet('instance');
-        };
     },
     afterEach: function() {
         this.$container.remove();
+    },
+    createBullet(options) {
+        return this.$container.dxBullet($.extend(true, {
+            tooltip: {
+                enabled: true
+            }
+        }, options)).dxBullet('instance');
     }
 };
 
-QUnit.module('Tooltip creating', environment);
+QUnit.module('Bullet tooltip', environment);
 
-QUnit.test('Enabled tooltip', function(assert) {
+QUnit.test('Tooltip constructor should accept valid parems when tooltip enabled', function(assert) {
```

Comment:
plz, check everywhere
```suggestion
QUnit.test('Tooltip constructor should accept valid params when tooltip is enabled', function(assert) {
```